### ISSUE TITLE
homebrew formula

### ISSUE THREAD

**Issue by cemo** [gchaincl/httplab#23]
It would be nice to be integrated into homebrew.

**Comment by gchaincl** [comment]
yep, that would be great, (un)fortunately I don't have a Mac, but I'll tag this issue with `help wanted`.

**Comment by m1ome** [comment]
@gchaincl i am on this one, currently creating formula.

**Comment by m1ome** [comment]
https://github.com/Homebrew/homebrew-core/pull/12351 waiting to PR to be merged.

**Comment by gchaincl** [comment]
Thanks for doing this @m1ome

**Comment by gwerbin** [comment]
It looks like the PR was rejected due to not having thorough enough post-install tests. Is there something better than `httplab -version`? It seems the Homebrew team is OK with that kind of test only if there isn't a more thorough alternative.

**Comment by gchaincl** [comment]
@gwerbin can you give me more context? What kind of test are they expecting to have?
PS: Never used brew before?

**Comment by gwerbin** [comment]
I'm not sure what they mean. You did the right thing by asking in that thread.

**Comment by gchaincl** [comment]
@m1ome any progress on this?

**Comment by gwerbin** [comment]
They're looking for some way (other than `httplab -version` to test that the installation went well.

**Comment by gchaincl** [comment]
Isn't that enough?

**Comment by gwerbin** [comment]
Evidently not, but if you can't find a better test that can be encoded in a Ruby script then your best bet is to make a case to the maintainer (the user "ilovezfs") that it's fine as-is. If not, I have an alternative tap (github.com/gwerbin/homebrew-tap) for software that I personally use, so I could host it there in the interim. Mind you, I'm not much of a FOSS maintainer myself so I can't guarantee it will always be up-to-date.

**Comment by gchaincl** [comment]
Thanks @gwerbin
